research/coplenet-pneumonia-lesion-segmentation/README.md

```diff
@@ -21,7 +21,14 @@ This research prototype is adapted from:
 
 To run the inference demo:
 
-- install the latest version of MONAI: `git clone https://github.com/Project-MONAI/MONAI` and `pip install -e .`.
+- Download and switch to MONAI 0.2.0 source code:
+```bash
+git clone https://github.com/Project-MONAI/MONAI
+cd MONAI
+git checkout 0.2.0
```

Comment:
1. Is it `releases/0.2.0` or `0.2.0`?
2. I think maybe it's easier to download 0.2.0 branch directly?
`git clone https://github.com/Project-MONAI/MONAI --branch releases/0.2.0`

Thanks.

Reply:
I think the current way is more clear, and I've checked it works fine